integration/mediation-tests/tests-other/src/test/java/org/wso2/carbon/esb/mediationstats/test/MediationStatEnableTestCase.java

```diff
@@ -50,11 +49,6 @@
 	@BeforeClass(alwaysRun = true)
 	public void setEnvironment() throws Exception {
 		super.init();
-		ServerConfigurationManager serverConfigurationManager =
-				new ServerConfigurationManager(new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
-		serverConfigurationManager.applyConfigurationWithoutRestart(new File(resourceFolderPath + "carbon.xml"));
-		serverConfigurationManager.applyConfiguration(new File(resourceFolderPath + "synapse.properties"));
```

Comment:
Isn't this synapse.properties file adding any configurations other than the default values?